web-wallet/src/routes/(welcome)/setup/+page.svelte

```diff
@@ -41,6 +41,10 @@
 	</div>
 </section>
 
+<footer>
+	<span>Web Wallet v{import.meta.env.APP_VERSION} ({import.meta.env.APP_BUILD_INFO})</span>
```

Comment:
> I would center the text and drop the font size to 12px, just because it looks better.

See @laremas's [design](https://www.figma.com/file/ImsgL6Xf0R31HFUIw2xxnw/Wallet-Extension-(FEB-08)?node-id=2784%3A21123&mode=dev).

There's no rush to merge though, so let's hear from him.

Reply:
Indeed, the design takes into account the settings button. In that case it makes sense to not have the version text centered.

Reply:
Hm. The version doesn't appear on the dashboard and the settings button is only available on the dashboard. So I am not 100% sure that the design is correct and that the text cannot be centered.

cc: @laremas

Reply:
By the way @nortonandreev and @deuch13 I don't mind having it centered now, and moved once we introduce the settings button: it's just that it doesn't bother me having it left aligned even without the button.

So @laremas: your pick.

Reply:
I don't mind about the centering as much, left-aligned is usually better for accessibility. But I think the font size should be smaller or opacity less (or both).